seed/static/seed/partials/about.html

```diff
@@ -8,6 +8,7 @@ <h1 translate>About SEED Platform™</h1>
 
                     <p translate>The Standard Energy Efficiency Data (SEED)™ Platform is a software application that helps organizations easily manage data on the energy performance of large groups of buildings. Users can combine data from multiple sources, clean and validate it, and share the information with others. The software application provides an easy, flexible, and cost-effective method to improve the quality and availability of data to help demonstrate the economic and environmental benefits of energy efficiency, to implement programs, and to target investment activity.</p>
                     <p><a href="http://energy.gov/eere/buildings/standard-energy-efficiency-data-platform" target="_blank" rel="noopener noreferrer" translate>More details</a></p>
+                    <p><a href="{$:: urls.static_url $}/Terms_of_Service.txt" target="_blank" rel="noopener noreferrer">View Terms of Service</a></p>
```

Comment:
This one didn't open for me 
![Screen Shot 2021-03-26 at 10 36 04 AM](https://user-images.githubusercontent.com/30608004/112664041-1dcc7100-8e1f-11eb-94eb-46601e4a875a.png)

Reply:
hmm, have you tried running collectstatic?